packages/editor-ui/src/components/NodeWebhooks.vue

```diff
@@ -68,8 +65,8 @@ export default mixins(
 		],
 		data () {
 			return {
-				isMinimized: true,
-				showUrlFor: 'Production',
+				isMinimized: false,
+				showUrlFor: 'Test URL',
```

Comment:
can we keep this as test? maybe even lowercased?
```suggestion
				showUrlFor: 'test',
```